src/sequelize/models/User.js

```diff
@@ -33,6 +33,11 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       defaultValue: 'user',
     },
+    status: {
```

Comment:
This should be a boolean field since we expect it to only have two states. You can name it either `active`/`verified`.

Reply:
I have implemented that.